fishy/engine/semifisher/fishing_event.py

```diff
@@ -32,9 +34,16 @@ class FishEvent:
     uid = None
     sound = False
 
+def _fishing_sleep(waittime, lower_limit = 16, upper_limit = 58):
```

Comment:
usually time in python is considered in seconds, if you want millisecs try to name ur variable with `_ms` in the end or mention it in doc string to improve code readability

Reply:
Okay. This is just to avoid hickups with randrange. Will call it *_ms then.